src/plat/pc99/machine/intel-vtd.c

```diff
@@ -249,7 +249,7 @@ static void vtd_process_faults(drhu_id_t i)
             address[0] = vtd_read32(i, fr_reg);
             reason = vtd_read32(i, fr_reg + 12) & FR_MASK;
 
-            printf("IOMMU: DMA %s page fault ", fault_type ? "read" : "write");
+            printf("IOMMU 0x%x: DMA %s page fault ", i, fault_type ? "read" : "write");
```

Comment:
It might be better to not put this change into the commit that does the renaming.

Reply:
Agreed it's better, but life's too short for that. The amount of hassle far outweighs the gain in this case, so I'd say just leave it this time. (I checked that `i` is always 32-bits, so it should be fine.)

Reply:
Sorry, but non-clean commits will cost you much more lifetime in the long run when you try to rebase or cherry-pick and run into conflict. Also, the amount of work doing this is rather trivial.